### ISSUE TITLE
Issue with tabbing

### ISSUE THREAD

**Issue by Loonyy** [DevLeoko/AdvancedBan#62]
_What version of AdvancedBan (`/AdvancedBan`) are you using?_
 Latest version from Spigot
 
_What kind of server do you have (`Bungeecord/single server`)?_
 Bungeecord
 
_What server version (/version) are you using?_
This server is running CraftBukkit version git-Spigot-7d78b81-f709362 (MC: 1.11.2) (Implementing API version 1.11.2-R0.1-SNAPSHOT)

_Please provide the EXACT steps required to reproduce the problem..._
Basically, when you start typing someone's name and then press tab for auto completion; it starts going through each person online's name in order, rather than completing the name.

When I had this plugin on my normal Spigot server, I could tab-complete just fine.

**Comment by dutchy1001** [comment]
I think there are some known issue's regarding to the tab completion. 
It's a matter of waiting for a new release with some other fixes as well.

**Comment by ZombieHDGaming** [comment]
Hi there! We have already found out about this issue, and it will be fixed in a newer release. For now though, I compiled AdvancedBan myself with the added fixes, which you can get in this thread:
https://github.com/DevLeoko/AdvancedBan/issues/44

**Comment by Loonyy** [comment]
Thank you!
